### ISSUE TITLE
Md5 with awd skeletal animation broadcast all have problems

### ISSUE THREAD

**Issue by SmileSFL** [Rajawali/Rajawali#1855]
MD5 animation broadcast, distortion of the arm and the awd skeletal animation broadcast, the body is deformed, do not know that you have seen no, there is no question of a version before, but this version is a problem now, I see let me think of the effect of matrix is got 180.

**Comment by ToxicBakery** [comment]
From #1857 

```
A few days ago, I am here to submit a question, is about the skeletal animation, MD5 format of skeletal animation, after completion of loading, play, the entire model of skeletal animation is twisted, its action also is wrong, and AWD skeletal animation is better, is the two arms of the animation is wrong, the problem can be solved? This is the second time I submit this problem, hope to be able to catch your eye, to modify this mistake.
```

**Comment by hasanonur** [comment]
Yes the body is still deformed. I think it is about the byte buffers . In v.09 we use shared library file "libbufferutil.so" to load it but it doesn't work anymore with API 27 as target version of minumum as Google wants. Please guys we really love this engine, recompile ."libbufferutil.so" with last NDK or improve mesh loaders :(

**Comment by zcZhangCheng** [comment]
> Yes the body is still deformed. I think it is about the byte buffers . In v.09 we use shared library file "libbufferutil.so" to load it but it doesn't work anymore with API 27 as target version of minumum as Google wants. Please guys we really love this engine, recompile ."libbufferutil.so" with last NDK or improve mesh loaders in the main of rajawali :(

How should I implement this solution specifically, I don't understand。 Thanks
